docs/content/testing/running-tests.md

```diff
@@ -0,0 +1,62 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../../README.md#recommended-use).
+
+-----
+
+# Running tests
+
+Run the following commands in your terminal from the root of the repository.
```

Comment:
What am I testing for? It's not clear. 
When do we recommend to run them? After every change? 

Maybe under the title have:
"Use these tests to make sure the <x, y, or z> runs correctly after you have modified them. Note that there are different tests for different functionality." 
*functionality might not be the right term -  there may be a better one.

Reply:
Does "running tests" mean "this doc is about how to run tests" or "this doc is about a type of test called 'running tests' "?

Reply:
Rephrasing this section based on your comments